src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Order.xml

```diff
@@ -111,6 +111,8 @@
         <attribute name="localeCode">
             <group>shop:order:account:read</group>
             <group>checkout:read</group>
+            <group>cart:update</group>
+            <group>order:read</group>
```

Comment:
As you are touching this area, can we introduce `shop:` prefix to order read serialization? Can be done in separate PR

Reply:
Ok I do it in new PR after merge this